src/main/java/org/simplify4u/plugins/keyserver/PGPKeysServerClient.java

```diff
@@ -326,7 +338,9 @@ private CloseableHttpClient buildClient(RoundRobinRouterPlaner planer) {
         final HttpClientBuilder clientBuilder = this.createClientBuilder();
 
         this.applyTimeouts(clientBuilder);
-        clientBuilder.setRoutePlanner(planer);
+        if (this.proxy == null) {
+            clientBuilder.setRoutePlanner(planer);
+        }
```

Comment:
I think it will be better implement RouterPlaner to support proxy like `org.apache.http.impl.conn.DefaultProxyRoutePlanner`

`setupProxy` should be call something her 

new router planer should be created about line 216/228

Reply:
chnaged that to DefaultProxyRoutePlanner